README.md

```diff
@@ -1,13 +1,7 @@
 GitLab Charm for Kubernetes Models
 ==================================
 
-To use, first install the framework using pip:
-
-```bash
-pip install -t lib/ https://github.com/canonical/operator
-```
-
-Then deploy the charm with an appropriate image resource:
+To use, deploy the charm with an appropriate image resource:
```

Comment:
Submodules aren't pulled in automatically when a repo is cloned.  It looks like we still need instructions to run `git submodule init` followed by `git submodule update`.